### PR TITLE
compiler-rt 17.0.2

### PR DESCRIPTION
**Pull request by h-vetinari**
Waiting for builds from https://github.com/conda-forge/clangdev-feedstock/pull/241

Otherwise no big changes

### CODE REVIEW

**Comment by h-vetinari** [comment]
Note: last rerender needed manual fixes due to https://github.com/conda-forge/conda-smithy/issues/1773

File: .azure-pipelines/azure-pipelines-win.yml [hunk 1]
```diff
@@ -35,52 +36,14 @@ jobs:
...
-
-    - script: |
-        call activate base
-        if EXIST LICENSE.txt (
-          copy LICENSE.txt "recipe\\recipe-scripts-license.txt"
-        )
-        conda.exe mambabuild "recipe" -m .ci_support\%CONFIG%.yaml --suppress-variables
-      displayName: Build recipe
+        call ".scripts\run_win_build.bat"
+      displayName: Run Windows build
```
Comment (jaimergp) [inline .azure-pipelines/azure-pipelines-win.yml:40]: Isn't this script missing?

Comment (h-vetinari) [reply .azure-pipelines/azure-pipelines-win.yml:40]: Possibly. I may have messed up what I manually picked into the rerender commit. I'll fix this once we have the respective clangdev builds.

**Comment by h-vetinari** [comment]
@conda-forge/compiler-rt, this is ready for review, PTAL. 🙏 

It should be nice and easy; I only documented the installation situation on windows a bit (we're installing the same files once into `%PREFIX%\lib` and once into `%PREFIX%\Library\lib`).

**Review by xhochy — APPROVED** [review]
LGTM

**Comment by h-vetinari** [comment]
OK, this one should be pretty routine.

**Pull request merged**